<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_M_2</class>
 <widget class="QWidget" name="Form_M_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel_M_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pengaturanButton_M_2">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium Cond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pengaturan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mataKuliahButton_M_2">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium Cond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mata Kuliah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="berandaButton_M_2">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Medium Cond</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Beranda</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoSiak_M_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>368</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
